rule-engine/rule-engine-components/pom.xml

```diff
@@ -35,7 +35,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <main.dir>${basedir}/../..</main.dir>
-        <aws.sdk.version>1.11.323</aws.sdk.version>
+        <aws.sdk.version>1.11.747</aws.sdk.version>
```

Comment:
This needs to be removed. We have the same in main pom.xml

Reply:
Please, remove from pull request file: ui/package-lock.json